### PR TITLE
common-instancetypes-periodics: Add initial job for release-1.1

### PR DESCRIPTION
**Pull request by lyarwood**
/cc @0xFelix 
/cc @brianmcarey 

**What this PR does / why we need it**:

For the time being this just syncs releases from release-1.0 to KubeVirt release-1.3 while we wait for the full v1.1.0 release of common-instancetypes.

This should keep these branches in line with the older release branches for now where we still sync release-1.0 releases.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml [hunk 1]
```diff
@@ -41,6 +41,42 @@ periodics:
...
+      base_ref: release-1.3
+    labels:
+      preset-github-credentials: "true"
+    spec:
+      containers:
+      - image: quay.io/kubevirtci/pr-creator:v20240305-cb764ec
+        env:
+          - name: COMMON_INSTANCETYPES_BRANCH
+            value: "release-1.0"
+          - name: KUBEVIRT_BRANCH
```
Comment (0xFelix) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:65]: I don't see SSP release-v0.21 here like the description suggests?

Comment (lyarwood) [reply github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:65]: Apologies I edited the commit message and forgot about the PR description. That release doesn't exist yet so we can't sync anything for now.

**Review by brianmcarey — COMMENTED** [review]
/approve

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Pull request merged**